tests/debugger/test_debugger.py

```diff
@@ -76,23 +78,95 @@ def check_info_endpoint():
     raise ValueError("Agent did not provide /v0.7/config endpoint")
 
 
-@released(java="1.19.3", php_appsec="?", python="?")
 @missing_feature(
     context.library == "java" and context.weblog_variant not in ["spring-boot", "uds-spring-boot"],
     reason="not supported",
 )
+@irrelevant(library="golang")
+@irrelevant(library="python")
+@irrelevant(library="ruby")
+@irrelevant(library="php")
+@irrelevant(library="nodejs")
+@irrelevant(library="cpp")
 @scenarios.debugger_method_probes_status
 class Test_Debugger_Method_Probe_Statuses:
     def test_method_probe_status(self):
-        check_probe_statuses(METHOD_PROBES)
+        expected_data = [
+            "logProbe-received",
+            "logProbe-installed",
+            "metricProbe-received",
+            "metricProbe-installed",
+            "spanProbe-received",
+            "spanProbe-installed",
+            "spanDecorationProbe-received",
+            "spanDecorationProbe-installed",
+        ]
+        validate_data(expected_data, [])
 
 
-@released(java="1.19.3", php_appsec="?", python="?")
 @missing_feature(
     context.library == "java" and context.weblog_variant not in ["spring-boot", "uds-spring-boot"],
     reason="not supported",
 )
+@irrelevant(library="golang")
+@irrelevant(library="python")
+@irrelevant(library="ruby")
+@irrelevant(library="php")
+@irrelevant(library="nodejs")
+@irrelevant(library="cpp")
 @scenarios.debugger_line_probes_status
 class Test_Debugger_Line_Probe_Statuses:
     def test_line_probe_status(self):
-        check_probe_statuses(LINE_PROBES)
+        expected_data = ["logProbe-installed", "metricProbe-installed", "spanDecorationProbe-installed"]
+        validate_data(expected_data, [])
+
+
+@missing_feature(
+    context.library == "java" and context.weblog_variant not in ["spring-boot", "uds-spring-boot"],
+    reason="not supported",
+)
+@irrelevant(library="golang")
```

Comment:
It mean that this feature will never be implement on those tracers ?

Reply:
I see what you mean. The `debugger` feature has already been implemented for the `python` library, but the tests haven't been verified yet. So, I changed it from `irrelevant` to `missing_feature` here f1c74f1161b080a18e872612f79030efafcc5274.

For all other libraries, it will take a very long time for this feature to be implemented, so I prefer to leave it as `irrelevant` for now.